qiskit/user_config.py

```diff
@@ -146,6 +146,39 @@ def read_config_file(self):
                 self.settings['num_processes'] = num_processes
 
 
+def set_config(key, value, section=None):
```

Comment:
I would also add an optional `filename` kwarg here too, just in case a user wants to write a new config file outside of what's configured.

Reply:
Does setting `QISKIT_SETTINGS` cover this use-case?

Reply:
I was thinking more if you want to create a new file separate the one currently loaded. `QISKIT_SETTINGS` only controls where the path to the file currently being used. So you could use this to adjust the path, but if you wanted to write to a different path from the file currently loaded this wouldn't work unless you did something like:

```python3
path_backup = os.get_environ('QISKIT_SETTINGS')
os.environ['QISKIT_SETTINGS'] = '/new/path'
qiskit.user_config.set_config('circuit_drawer', 'latex')
os.environ['QISKIT_SETTINGS'] = path_backup
```

which seems a bit clunky.

Reply:
I tried adding a filename kwarg (optional), but `QISKIT_SETTINGS` has to be set because Qiskit re-renders the config from the default config file or the `QISKIT_SETTINGS` file whenever there is an import from qiskit. I thought, setting the env var without the user adding it might create some confusion. Can you suggest a way of doing it?

Reply:
@mtreinish Is the idea to create the function `set_config` with filename kwarg to be able to create multiple config files and use `QISKIT_SETTINGS` as a switch between config files?